### DISCUSSION TITLE
Which Whisper is the best?

### DISCUSSION THREAD

**Discussion by ioudove** [openai/whisper#2267] [Q&A] [answered]
I found two upgraded versions of Whisper on the internet, which have been modified to a certain extent based on the original Whisper code. I don't understand coding. Could you please tell me which version of Whisper is the best right now? I mean the one with the highest accuracy.

Here are the two upgraded versions of Whisper I found online:

1、https://github.com/SYSTRAN/faster-whisper
2、https://github.com/Vaibhavs10/fast-whisper-finetuning

If you have a better upgraded version of Whisper, please share the link with me. Thank you!

**Comment by phineas-pta** [comment, marked as answer]
they're called "faster" not "better", so u get the idea

**Reply by ioudove** [reply]
OK， thank you

**Comment by sin2pie** [comment]
None of the faster versions are as accurate in practice. Try to avoid quantization if you can.  It's better to train smaller models than to quantize larger ones in my experience. Statements like these "LoRA performs on-par or better than fine-tuning in model quality" are 100% false unless they mean performs on-par according to their evaluation metrics.

**Reply by ioudove** [reply]
OK Thank you ：D
